heron/packing/tests/java/com/twitter/heron/packing/binpacking/FirstFitDecreasingPackingTest.java

```diff
@@ -271,17 +303,17 @@ public void testCompleteRamMapRequested() throws Exception {
     Assert.assertEquals(packingPlanExplicitRamMap.containers.size(), 1);
 
     Assert.assertEquals(Math.round(7 * instanceCpuDefault
-            + 10 * (7 * instanceCpuDefault) / 100
+            + DEFAULT_CONTAINER_PADDING * (7 * instanceCpuDefault) / 100
             + instanceCpuDefault),
         (long) packingPlanExplicitRamMap.resource.cpu);
 
     Assert.assertEquals(4 * spoutRam + 3 * boltRam
-            + 10 * (4 * spoutRam + 3 * boltRam) / 100
+            + DEFAULT_CONTAINER_PADDING * (4 * spoutRam + 3 * boltRam) / 100
```

Comment:
Instead of 4 and 3 could we use `spoutParallelism` and `boltParallelism` here (and elsewhere) respectively. Ideally we define ints with a member variable that states what it represents and then we only use the members, not the ints.